### PR TITLE
Add some tests

### PR DESCRIPTION
**Pull request by gluhar2006**

### CODE REVIEW

File: tests/test_estimate_emotions.py [hunk 1]
```diff
@@ -46,10 +46,12 @@ def test_estimate_emotions_as_dict(self):
         """
         Test emotion estimator 'asDict' method
         """
-        faceDetection = self.detector.detectOne(EMOTION_IMAGES["fear"])
-        warp = self.warper.warp(faceDetection)
-        emotionDict = self.emotionEstimator.estimate(warp.warpedImage).asDict()
-        self.validate_emotion_dict(emotionDict)
+        for emotion in ALL_EMOTIONS:
+            with self.subTest(image_emotion=emotion):
```
Comment (matemax) [inline tests/test_estimate_emotions.py]: For what? It is  test of a method asDict only/

File: tests/test_estimate_eyes.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+        self.assert_eyes_reply(eyesDict)
+
+    def test_estimate_open_eyes(self):
+        """
+        Test eye estimator with face with opened eyes
+        """
+        faceDetection = self.detector.detectOne(OPEN_EYES_IMAGE)
+        warp = self.warper.warp(faceDetection)
+        landMarks5Transformation = self.warper.makeWarpTransformationWithLandmarks(faceDetection, "L5")
+        eyesDict = self.eyeEstimator.estimate(landMarks5Transformation, warp.warpedImage).asDict()
```
Comment (matemax) [inline tests/test_estimate_eyes.py]: You check method asDict,  this method is `consequence` of estimation result. see https://github.com/matemax/lunasdk/blob/master/tests/test_qualityestimator.py#L24

File: tests/test_estimate_eyes.py [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+    def test_estimate_closed_eyes(self):
+        """
+        Test eye estimator with face with closed eyes
+        """
+        faceDetection = self.detector.detectOne(CLOSED_EYES_IMAGE)
+        warp = self.warper.warp(faceDetection)
+        landMarks5Transformation = self.warper.makeWarpTransformationWithLandmarks(faceDetection, "L5")
+        eyesDict = self.eyeEstimator.estimate(landMarks5Transformation, warp.warpedImage).asDict()
+        assert eyesDict["left_eye"]["state"] == "closed"
```
Comment (matemax) [inline tests/test_estimate_eyes.py]: stop use `asDict`

**Review by matemax — APPROVED** [review]

File: tests/test_estimate_emotions.py (hunk 1 above)
Comment (gluhar2006) [reply tests/test_estimate_emotions.py]: removed

File: tests/test_estimate_eyes.py (hunk 2 above)
Comment (gluhar2006) [reply tests/test_estimate_eyes.py]: asDict has been removed

File: tests/test_estimate_eyes.py (hunk 3 above)
Comment (gluhar2006) [reply tests/test_estimate_eyes.py]: asDict has been removed

**Pull request merged**